src/components/Header.vue

```diff
@@ -1,40 +1,68 @@
-    },
+      components: {CartFlyout},
+      name: 'Header',
+      props: {
+          mobileMenuOpen: Boolean,
+          cartOpen: Boolean,
+          totalQuantity: {
+              type: Number,
+              default: 0,
+          },
+          cartItems: {
+            type: Array as () => CartItemInterface[],
+            required: true,
+          },
+      },
+      setup(props, context) {
+        const handleRemoveFromCart = (itemId: number) => {
+          console.log('remove from cart', itemId);
+          context.emit('remove-from-cart', itemId);
+        };
+
+        return {
+          handleRemoveFromCart,
+        };
+      },
     });
 </script>
 
 <template>
-    <header class="py-5 px-6 flex justify-between items-center">
-      <div class="flex justify-between items-center gap-4">
-        <div class="md:hidden">
-          <button class="flex justify-center items-center" @click="$emit('toggle-mobile-menu')">
-            <img src="../assets/icon-menu.svg" alt="menu" />
-          </button>
-        </div>
-        <img class="self-center pb-1" src="../assets/logo.svg" alt="sneakers" />
+    <header class="py-5 md:py-0 px-6 md:px-0 flex justify-between items-center border-b-neutral-grey border-b-2 relative">
+      <div class="flex items-center">
+        <button class="md:hidden" @click="$emit('toggle-mobile-menu')">
+          <img src="../assets/icon-menu.svg" alt="menu" />
+        </button>
+        <img class="self-center pb-1 ml-4 md:ml-0" src="../assets/logo.svg" alt="sneakers" />
+        <nav class="hidden md:flex justify-between items-center gap-8 pl-14 text-neutral-dark-grayish-blue">
+          <a href="#" class="hover:text-[#1D2026] border-b-4 border-b-transparent hover:border-primary-orange py-10">Collections</a>
+          <a href="#" class="hover:text-[#1D2026] border-b-4 border-b-transparent hover:border-primary-orange py-10">Men</a>
+          <a href="#" class="hover:text-[#1D2026] border-b-4 border-b-transparent hover:border-primary-orange py-10">Women</a>
+          <a href="#" class="hover:text-[#1D2026] border-b-4 border-b-transparent hover:border-primary-orange py-10">About</a>
+          <a href="#" class="hover:text-[#1D2026] border-b-4 border-b-transparent hover:border-primary-orange py-10">Contact</a>
+        </nav>
+      </div>
+
+
+      <div class="absolute right-[-76px] top-[94px] w-[360px] hidden md:block">
+          <CartFlyout :isOpen="cartOpen" :cartItems="cartItems" @remove-item="handleRemoveFromCart" />
       </div>
-  
-      <div class="flex gap-5">
+      <div class="flex items-center gap-5 md:gap-12 ">
         <button class="relative" @click="$emit('toggle-cart')">
           <img src="../assets/icon-cart.svg" alt="cart" />
-          <span v-if="totalQuantity > 0"
-            class="absolute top-[-3px] right-[-5px] text-white text-[10px] font-bold bg-primary-orange rounded-full w-5 flex items-center justify-center">
+          <span
+            v-if="totalQuantity > 0"
```

Comment:
It would be better to use a computed property to calculate the quantity and display it in the template. This makes the code more readable and easier to understand.